src/analyze/analyze.c

```diff
@@ -2470,6 +2477,7 @@ static int help(int argc, char *argv[], void *userdata) {
                "  timestamp TIMESTAMP...     Validate a timestamp\n"
                "  timespan SPAN...           Validate a time span\n"
                "  security [UNIT...]         Analyze security of unit\n"
+               "  inspect-elf FILE...        Parse and print ELF package metadata\n"
```

Comment:
Hmm, why is this here, and not in coredumpctl? Wouldn't it be more natural to say 'coredumpctl inspect-elf ...`?

Reply:
request from @poettering - the idea is that we don't just look at corefiles and not just at package metadata, but at all ELF files and also print generic stuff (elf type/architecture, build-id), and then going forward we can add more stuff if it is useful to do so

Reply:
OK. I don't necessarily agree, but it doesn't make sense to move it back and forth.